templates/react_ts/build.js

```diff
@@ -0,0 +1,83 @@
+//
+// This is an example build script for Socket Runtime
+// When you run 'ssc build', this script (node build.js) will be run
+//
+import fs from 'node:fs'
+import path from 'node:path'
+
+import esbuild from 'esbuild'
+
+const cp = async (a, b) =>
+  fs.promises.cp(path.resolve(a), path.join(b, path.basename(a)), {
+    recursive: true,
+    force: true
+  })
+
+async function main() {
+  const prod = process.argv.find((s) => s.includes('--prod'))
+
+  const params = {
+    entryPoints: ['src/index.tsx'],
+    format: 'esm',
+    bundle: true,
+    minify: !!prod,
+    sourcemap: !prod,
```

Comment:
Should this be `!!prod` as well?

Reply:
I think yes , it should be enabled based on this --prod flag . I just checked the esbuild docs and as far I understand them , we should enable sourcemaps, if generated code is sufficiently different from original code (e.g. original code is TypeScript or we enabled [minification]. ) I will change this to ```sourcemap: !!prod ```

Reply:
@chicoxyzzy Is there any reason not to enable `sourcemap`?

Reply:
sourcemap should be always enabled for dev, I think we can set it to `true`, but `sourcemap: !!prod` is definitely incorrect

Reply:
I'll change it to ```true``` then.

Reply:
actually web inspector is disabled in prod iirc, so there's no reason to ship useless files. `sourcemap: !prod` was a correct value